cmdeploy/src/cmdeploy/remote/rshell.py

```diff
@@ -0,0 +1,17 @@
+from subprocess import CalledProcessError as ShellError
```

Comment:
because the module is about running shell commands, and it's not neccessary to leak that it's subprocess.CalledProcessError ... maybe not neccessary but also not hurting i'd say.